Docs/source/running_cpp/parameters.rst

```diff
@@ -1023,6 +1023,11 @@ Numerics and algorithms
     Note that the implementation in WarpX is more efficient when these 3 numbers are equal,
     and when they are between 1 and 3.
 
+* ``interpolation.match_shape_factors`` (`0` or `1` ; default: `0`)
```

Comment:
Could you rename this to `interpolation.galerkin_scheme` (`0` or `1`) with `1` being the one that has lower order in some direction? This is the name that is used in this paper: https://www.sciencedirect.com/science/article/pii/S0021999113002556

Also, in the documentation, could be a bit more precise and say that this uses ``interpolation.nox-1`` instead of ``interpolation.nox`` when gathering ``E_x``, with similar changes in y and z, and that this has better energy-conserving properties. Could you also add a reference to the papers that are cited https://www.sciencedirect.com/science/article/pii/S0021999113002556 (reference 18 to 20)?

Finally, when describing the default, I think that the default should be `0` when running with a nodal grid, and `1` otherwise. (i.e. same `l_lower_order_in_v`. In fact I would suggest to rename `l_lower_order_in_v` as `galerkin_interpolation` ; see below.)